docs/docs/01-start-here/02-getting-started.md

```diff
@@ -1,10 +1,27 @@
 ---
-id: installation
-title: Installation
-keywords: [Wing installation, installation, Wing toolchain]
+id: getting-started
+title: Getting Started
+keywords: [Getting started, Wing installation, installation, Wing toolchain]
 slug: /
 ---
 
+## Welcome
```

Comment:
@hasanaburayyan I made a few edits. Please verify that this renders nicely!

Reply:
Ah shoot I thought I already resolved this